doc/api/net.md

```diff
@@ -281,6 +281,26 @@ added: v0.1.90
 
 Emitted when the server has been bound after calling [`server.listen()`][].
 
+### Event: `'drop'`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+When the number of connections reaches the threshold of `server.maxConnections`,
+the server will drop new connections and emit `'drop'` event instead. If it is a
+TCP server, the arguments is as follow, otherwise return `undefined`.
```

Comment:
```suggestion
When the number of connections reaches the threshold of `server.maxConnections`,
the server will drop new connections and emit `'drop'` event instead. If it is a
TCP server, the argument is as follows, otherwise, the argument is `undefined`.
```

Reply:
Done !